src/turbomind/models/llama/LlamaBatch.cc

```diff
@@ -1522,6 +1552,14 @@ bool LlamaBatch::Forward(GenerationState& g)
             }
         }
 
+        std::shared_ptr<MultimodalRope> mrope_sptr;
```

Comment:
`std::optional`, or even `std::vector` is better than explicit dynamic allocation